src/box/lua/load_cfg.lua

```diff
@@ -54,6 +54,9 @@ local default_cfg = {
     checkpoint_count    = 2,
     worker_pool_threads = 4,
     replication_timeout = 1,
+    reports_enabled     = true,
```

Comment:
All configuration variable names should be in singular form. Please check the names with Elena, as well as all error messages.